setup.py

```diff
@@ -140,5 +140,5 @@ def cuda_ext_helper(name, sources, extra_cuda_flags):
     ext_modules=ext_modules,
     package_data={'fastfold': ['model/fastnn/kernel/cuda_native/csrc/*']},
     cmdclass={'build_ext': BuildExtension} if ext_modules else {},
-    install_requires=['einops', 'colossalai'],
+    install_requires=['einops'],
```

Comment:
Just confirm that colossalai will be installed if user install fastfold by `python setup.py install` or `pip`, weather colossalai wi